src/middleware/kafkaProducerSet.js

```diff
@@ -0,0 +1,57 @@
+import logger from 'winston'
+
+import {KafkaProducer} from './kafkaProducer.js'
+
+export class KafkaProducerSet {
+  static kafkaSet = []
```

Comment:
Let's change this to "{}" instead of array, to allow access by key (string computed from clientId + broker). This prevent us from doing loops to search and thus enhance performance.

Reply:
cool done 👍🏼